sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-offering-details-report.sql

```diff
@@ -0,0 +1,72 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Offering_Details_Report',
+    'SELECT
+      institution_locations.institution_code AS "Institution Location Code",
+      education_programs.program_name AS "Program",
+      education_programs.sabc_code AS "SABC Program Code",
+      education_programs_offerings.offering_name AS "Offering Name",
+      education_programs_offerings.year_of_study AS "Year of Study",
+      education_programs_offerings."offering_intensity" AS "Offering Intensity",
+      education_programs_offerings.course_load AS "Course Load",
+      education_programs_offerings.offering_delivered AS "Delivery Type",
+      education_programs_offerings.has_offering_wil_component AS "WIL Component",
+      education_programs_offerings.offering_wil_type AS "WIL Component Type",
+      education_programs_offerings.study_start_date AS "Start Date",
+      education_programs_offerings.study_end_date AS "End Date",
+      NOT education_programs_offerings.lacks_study_breaks AS "Has Study Breaks",
+      COALESCE(education_programs_offerings.actual_tuition_costs,
+      0) AS "Actual Tuition",
+      COALESCE(education_programs_offerings.program_related_costs,
+      0) AS "Program Related Costs",
+      COALESCE(education_programs_offerings.mandatory_fees,
+      0) AS "Mandatory Fees",
+      COALESCE(education_programs_offerings.exceptional_expenses,
+      0) AS "Exceptional Expenses",
+      education_programs_offerings.offering_type AS "Public Offering",
```

Comment:
"Public Offering" must be converted to Yes/No or the column name should be changed to "Offering Type".
I would suggest the other with business is ok.